src/pages/billing/List/ProjectListItem.ts

```diff
@@ -17,66 +17,95 @@ import { BillingProjectActions } from 'src/pages/billing/List/BillingProjectActi
 import { BillingProject } from 'src/pages/billing/models/BillingProject'
 
 
-const styles = {
-  projectListItem: selected => {
+const listItemStyle = (selected, hovered) => {
+  const style = {
+    ...Style.navList.itemContainer(selected),
+    ...Style.navList.item(selected),
+    ...(selected ? { backgroundColor: colors.dark(0.1) } : {}),
+    paddingLeft: '2rem'
+  }
+  if (hovered) {
     return {
-      ...Style.navList.itemContainer(selected),
-      ...Style.navList.item(selected),
-      ...(selected ? { backgroundColor: colors.dark(0.1) } : {}),
-      paddingLeft: '3rem'
+      ...style,
+      ...Style.navList.itemHover(selected)
```

Comment:
Hover state now has to happen on the parent div instead of the link, as I moved the delete action outside of the link.